### PR TITLE
Updated Contributing.rst    Solves #543

### PR DESCRIPTION
**Pull request by rachittshah**
Changed make html to sphinx-build -b html source build in line 34,as discussed over the EinsteinPy IRC.

### CODE REVIEW

**Comment by JeS24** [comment]
@godslayer201 Is there any related issue open?

**Comment by godslayer201** [comment]
No, I think not. Should I open a corresponding issue?

**Comment by JeS24** [comment]
@godslayer201 Please do so. Something like "`make html` does not work on Windows".

**Comment by godslayer201** [comment]
Will do. 


Will add relevant details here as well.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ To build the docs, you must first create a development environment (see
 below) and then in the ``docs/`` directory run::
 
     $ cd docs
-    $ make html
+    $ sphinx-build -b html source build
```
Comment (JeS24) [inline CONTRIBUTING.rst:34]: @godslayer201 
Make this `sphinx-build -b html source build/html`.

Later, in the same file, change `python -m http.server`
to `python -m http.server 1000 --bind 127.0.0.1`, as `python -m http.server` can lead to `OSError: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions` on Windows (It's not a UAC issue. Something's broken in `http.server`).

Finally, change "And point your browser to http://0.0.0.0:8000." to "And point your browser to http://127.0.0.1:1000.".

Comment (godslayer201) [reply CONTRIBUTING.rst:34]: Understood. 

Will update my PR.

Comment (shreyasbapat) [reply CONTRIBUTING.rst:34]: Hey @godslayer201 Any updates on this?

Comment (godslayer201) [reply CONTRIBUTING.rst:34]: Hey Shreyas,

I'm not working on it at the moment.

I would finish the PR by Friday

**Comment by JeS24** [comment]
Completed in #615.

**Pull request closed without merging**